nntrainer/cl_context.h

```diff
@@ -215,8 +215,10 @@ class ClContext : public Context {
    * @param kernel_name kernel name
    * @return std::shared_ptr<opencl::Kernel>
```

Comment:
Can you also add a description here?

```suggestion
   * @param compiler_options compiler options for the OpenCL kernel compiler.
   * @return std::shared_ptr<opencl::Kernel>
```